### PR TITLE
[RSML] two new end-to-end labs featuring Cloud Composer

### PR DESCRIPTION
**Pull request by jonesevan**
one lab on Google Cloud Function triggering workflows inspired by:
https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/cloud-composer-examples/composer_dataflow_examples/simple_load_dag.py

a second lab on WALS + Airflow for ML inspired by:
https://github.com/GoogleCloudPlatform/tensorflow-recommendation-wals/tree/master/airflow/deploy

### CODE REVIEW

**Comment by lakshmanok** [comment]
Looks like this PR includes

courses/machine_learning/deepdive/10_recommend/labs/endtoend/data/recommendation_events.csv

Please put data of this size on GCS and in the github repo. I'm closing this PR so that the repo size is manageable.  Please open a new PR without the data

File: courses/machine_learning/deepdive/10_recommend/endtoend/airflow/airflow.cfg [hunk 1]
```diff
@@ -0,0 +1,351 @@
+[core]
+# The home folder for airflow, default is ~/airflow
+airflow_home = /Users/lramsey/work/tfrec/rec_serve/airflow
+
+# The folder where your airflow pipelines live, most likely a
+# subfolder in a code repository
+# This path must be absolute
+dags_folder = /Users/lramsey/work/tfrec/rec_serve/airflow/dags
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/endtoend/airflow/airflow.cfg]: We want to probably remove all these absolute path mentions.

File: courses/machine_learning/deepdive/10_recommend/endtoend/airflow/airflow.cfg [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+# Whether to load the examples that ship with Airflow. It's good to
+# get started, but you probably want to set this to False in a production
+# environment
+load_examples = True
+
+# Where your Airflow plugins are stored
+plugins_folder = /Users/lramsey/work/tfrec/rec_serve/airflow/plugins
+
+# Secret key to save connection passwords in the db
+fernet_key = Oi7UoO_f2dzqhTM3Eb3Xg9cAbz0VubC_SMc5GHwZP14=
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/endtoend/airflow/airflow.cfg]: Probably also don't want to save this key to the repo

File: courses/machine_learning/deepdive/10_recommend/endtoend/airflow/unittests.cfg [hunk 3]
```diff
@@ -0,0 +1,61 @@
+[core]
+unit_test_mode = True
+airflow_home = /Users/lramsey/work/tfrec/rec_serve/airflow
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/endtoend/airflow/unittests.cfg]: absolute paths here as well

File: courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/airflow.cfg [hunk 4]
```diff
@@ -0,0 +1,351 @@
+[core]
+# The home folder for airflow, default is ~/airflow
+airflow_home = /Users/lramsey/work/tfrec/rec_serve/airflow
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/airflow.cfg]: absolute path here which needs to be removed

File: courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/airflow.cfg [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+# Whether to load the examples that ship with Airflow. It's good to
+# get started, but you probably want to set this to False in a production
+# environment
+load_examples = True
+
+# Where your Airflow plugins are stored
+plugins_folder = /Users/lramsey/work/tfrec/rec_serve/airflow/plugins
+
+# Secret key to save connection passwords in the db
+fernet_key = Oi7UoO_f2dzqhTM3Eb3Xg9cAbz0VubC_SMc5GHwZP14=
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/airflow.cfg]: also need to remove this key

File: courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/unittests.cfg [hunk 6]
```diff
@@ -0,0 +1,61 @@
+[core]
+unit_test_mode = True
```
Comment (alexhanna) [inline courses/machine_learning/deepdive/10_recommend/labs/endtoend/airflow/unittests.cfg]: absolute directory to be removed

**Pull request closed without merging**